### ISSUE TITLE
[FEATURE] Ability to configure plugin to use aliased links

### ISSUE THREAD

**Issue by jamesx4** [Dyldog/luhman-obsidian-plugin#33]
# Requested Feature
Ability to configure plugin to use aliased links

## Relates To
Even though I want to use Luhman UIDs (1a, 1a1, 1a2, etc) for my note filenames, I may not want those UIDs in my links between parent/child or sibling notes.  For me this creates a cleaner, more readable note in the note body itself while still allowing me to see context via the Luhman UID in the File Explorer. 

## Solution
I would love to have a plugin setting like _Use Aliased Links_ that would display just the title w/o the UID and separator in the links between parent/child or sibling notes.

For example,  with setting like _Use Aliased Links_ enabled, doing Create Child w/title "Child ABC" might create note _1a1 Child ABC.md_, but the link to child would be of the form <[[1a1 Child ABC|Child ABC]]>.  And the link placed on the new child back to the parent note would be <[[1a Parent Note|Parent Note]]>. 

## Considerations
This option would not effect *aliases:* property in the frontmatter, as that seems to be something someone would add explicitly if needed.

**Comment by jvanz** [comment]
Oh, I've just created the #34 asking for a similar thing. My bad.

**Comment by Eforen** [comment]
I agree that this is ultimatly desirable and is something I had planned to address later but I do not want to just see this as a boolean in the settings I also want an option under making the files in the firstplace that overrides the default behavior.

How I was planning to impliment it is set the default in the settings and then have a checkbox on the make new note screen or the link to note screen that defaults to whatever the default is that way you can change it whenever.

Aditionally I was likely going to make it so that there are multiple defaults one for each situation.

- Create New Child Note
- Create New Child Note from Selected Text
- Create New Sibling Note
- Create New Sibling Note from Selected Text
- Insert Link to Zettel Note

**Comment by jamesx4** [comment]
One other thought...I mentioned that I wouldn't want this to do anything with the aliases property in the frontmatter, but now I'm not so sure.  Been playing with that and it actually works really nice to add the title of my note to the aliases.  Then when I'm linking to it from disparate notes Obsidian will automatically use the alias in linking to a note with filename consisting of <uid><separator><title>.  Works quite well.  Perhaps, it would make sense to have an option to add title to aliases.
